docs/Using-EthSigner/Getting-Started.md

```diff
@@ -31,14 +31,14 @@ option set to `8590`.
 
 Create a text file containing the password for the V3 Keystore key file to be created (for example, `passwordFile`). 
 
-!!! attention "The password text file must not contain any other character than those used in your password"
-    EthSigner will read password file as binary and any character in it will be considered being part
+!!! attention "Password text file must not contain characters other than those used in your password"
+    EthSigner reads the password file as binary and any character in the file is considered part
```

Comment:
s/`character in the file is`/`characters in the file are` - plural?

Reply:
Already fixed